manual_tests/ch32838_test/readme.md

```diff
@@ -0,0 +1,9 @@
+## ch32838 Manual Test
+
+Manual test to run on the co-packing algorithm:
+ 
+**Test**
+cd ../../..
```

Comment:
I like reading/ writing tests best as

```
1) Goal of the test (if small) or the step in the test (if complex):
What is the target things to do 
  `example_or_template of command`
   Expected output 
````
So like
```
**Test**
1) Command line args match nautilus design 
This is to check the json file matches what nautilus generates 
`pack3d -input_config_json_filename manual_tests/ch32838_test/dense_input.json -output_packing_json_filename manual_tests/ch32838_test/output/output` 
This will output a $BAZ into the same directory  To check it,  open in $FOO program and look for $BAR 
```
But all those variables should be exact values/names to run the manual test.